llvm/lib/Passes/PassRegistry.def

```diff
@@ -385,6 +385,7 @@ FUNCTION_PASS("lower-allow-check", LowerAllowCheckPass())
 FUNCTION_PASS("lower-atomic", LowerAtomicPass())
 FUNCTION_PASS("lower-constant-intrinsics", LowerConstantIntrinsicsPass())
 FUNCTION_PASS("lower-expect", LowerExpectIntrinsicPass())
+FUNCTION_PASS("lower-conditional-store", LowerConditionalStoreIntrinsicPass())
```

Comment:
Should probably be named the same as the `DEBUG_TYPE` (which is `lower-cond-store-intrinsic`), or the `DEBUG_TYPE` should be renamed.